### PR TITLE
Quim/dashboard page docs

### PR DESCRIPTION
**Pull request by joaquimgamero**
I've added the dashboard page in the docs with a couple of screenshots and an explanation of the overall feature and a more detailed unscrambling of the page's sections.

Let me know if you think we can improve the text, since I went for a gut feeling on what a good overall explication, or even if you think it is lacking something important!

### CODE REVIEW

File: docs/dashboard/overview.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Dashboard Overview
+headline: 'Dashboard'
+description: 'Keep track of the workflows ran in Tower'
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:4]:
```suggestion
description: 'View and manage workflow runs in Tower'
```

File: docs/dashboard/overview.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: Dashboard Overview
+headline: 'Dashboard'
+description: 'View and manage workflow runs in Tower'
+---
+
+!!! note 
+    This feature is available from Tower v.22.3X onwards.
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:8]:
```suggestion
    This feature is available from Tower v.22.3.
```

Comment (joaquimgamero) [reply docs/dashboard/overview.md]: That sounds better, good catch!

File: docs/dashboard/overview.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+headline: 'Dashboard'
+description: 'View and manage workflow runs in Tower'
+---
+
+!!! note 
+    This feature is available from Tower v.22.3.
+
+## Overview
+
+Since Tower v22.3.X we have added to Tower the **Dashboard** page, which helps you keep track of where all you accessible workflow executions have been submitted. You will be able to access it via your personal menu in the top right corner. Click your avatar and select "Dashboard".
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:12]:
```suggestion
From version 22.3, Tower contains a **Dashboard** page that displays the details of workflow executions in your Workspace. The dashboard is accessed from the user menu in the top right corner. Click your avatar, then select "Dashboard".
```

File: docs/dashboard/overview.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+![](_images/dashboard_hero.png)
+
+The page is split into two main areas:
+
+## Filters and summary
+
+The drop-down lists at the top of the dashboard page allow you to filter the workflows displayed by personal Workspace, user Organizations, and time.
+
+Below these filters there is a section with a card per every possible workflow status, which yields a summary regarding the state of the total workflows.
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:22]:
```suggestion
Below the filtering options, a summary of total workflows is displayed using cards for each workflow status.
```

File: docs/dashboard/overview.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+headline: 'Dashboard'
+description: 'View and manage workflow runs in Tower'
+---
+
+!!! note 
+    This feature is available from Tower v.22.3.
+
+## Overview
+
+From version 22.3, Tower contains a **Dashboard** page that displays the details of workflow executions in your Workspace. The dashboard is accessed from the user menu in the top right corner. Click your avatar, then select "Dashboard".
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:12]:
```suggestion
From version 22.3, Tower contains a **Dashboard** page that displays the details of workflow runs in your Workspace. The dashboard is accessed from the user menu in the top right corner. Click your avatar, then select "Dashboard".
```

File: docs/dashboard/overview.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+## Filters and summary
+
+The drop-down lists at the top of the dashboard page allow you to filter the workflows displayed by personal Workspace, user Organizations, and time.
+
+Below the filtering options, a summary of total workflows is displayed using cards for each workflow status.
+
+## Workflows per organization
+
+Below the filters, you will be able to see one card for every organization you have access to (or a single one with your "Personal Workspace" if you have selected this option in the filter above).
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:26]:
```suggestion
Below the filters, cards containing workflow run totals are displayed. Depending on the options you selected in the drop-down above, the cards are filtered for each organization you have access to, or your personal Workspace.
```

File: docs/dashboard/overview.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+The page is split into two main areas:
+
+## Filters and summary
+
+The drop-down lists at the top of the dashboard page allow you to filter the workflows displayed by personal Workspace, user Organizations, and time.
+
+Below the filtering options, a summary of total workflows is displayed using cards for each workflow status.
+
+## Workflows per organization
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:24]:
```suggestion
## Workflow runs per organization
```

File: docs/dashboard/overview.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
+---
+title: Dashboard Overview
+headline: 'Dashboard'
+description: 'View and manage workflow runs in Tower'
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:4]:
```suggestion
description: 'View run status overview in Tower'
```

File: docs/dashboard/overview.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+headline: 'Dashboard'
+description: 'View run status overview in Tower'
+---
+
+!!! note 
+    This feature is available from Tower v.22.3.
+
+## Overview
+
+From version 22.3, Tower contains a **Dashboard** page that displays the details of workflow runs in your Workspace. The dashboard is accessed from the user menu in the top right corner. Click your avatar, then select "Dashboard".
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:12]:
```suggestion
From version 22.3, Tower contains a **Dashboard** page that displays the details of runs in your Organizations and Workspaces. The dashboard is accessed from the user menu in the top right corner. Click your avatar, then select "Dashboard".
```

File: docs/dashboard/overview.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+From version 22.3, Tower contains a **Dashboard** page that displays the details of runs in your Organizations and Workspaces. The dashboard is accessed from the user menu in the top right corner. Click your avatar, then select "Dashboard".
+
+![](_images/dashboard_hero.png)
+
+The page is split into two main areas:
+
+## Filters and summary
+
+The drop-down lists at the top of the dashboard page allow you to filter the workflows displayed by personal Workspace, user Organizations, and time.
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:20]:
```suggestion
The drop-down lists at the top of the dashboard page filter total runs by your personal Workspace, the Organizations you have access to, and time range.
```

File: docs/dashboard/overview.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+![](_images/dashboard_hero.png)
+
+The page is split into two main areas:
+
+## Filters and summary
+
+The drop-down lists at the top of the dashboard page filter total runs by your personal Workspace, the Organizations you have access to, and time range.
+
+Below the filtering options, a summary of total workflows is displayed using cards for each workflow status.
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:22]:
```suggestion
Below the filters, a summary of total runs is shown by status.
```

File: docs/dashboard/overview.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+From version 22.3, Tower contains a **Dashboard** page that provides an overview of runs in your organizations and personal workspace at a glance. The dashboard is accessed from the user menu in the top right corner. Click your avatar, then select "Dashboard".
+
+![](_images/dashboard_hero.png)
+
+The page is split into two main areas:
+
+## Filters and summary
+
+The drop-down lists at the top of the dashboard page filter total runs by your personal Workspace, the Organizations you have access to, and time range.
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:20]:
```suggestion
The drop-down lists at the top of the dashboard page filter total runs by your personal workspace, the organizations you have access to, and time range.
```

File: docs/dashboard/overview.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+The page is split into two main areas:
+
+## Filters and summary
+
+The drop-down lists at the top of the dashboard page filter total runs by your personal workspace, the organizations you have access to, and time range.
+
+Below the filters, a summary of total runs is shown by status.
+
+## Workflow runs per organization
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:24]:
```suggestion
## Runs per organization
```

File: docs/dashboard/overview.md [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+
+The drop-down lists at the top of the dashboard page filter total runs by your personal workspace, the organizations you have access to, and time range.
+
+Below the filters, a summary of total runs is shown by status.
+
+## Runs per organization
+
+Below the cards displaying total runs by status, run totals are filtered by each organization or your personal workspace. Filtering depends on what you selected in the drop-down options near the top of the page. 
+
+Every card consists of a row for every workspace you have access to inside this organizations, and at the same time a column for every possible workflow status, with a total column at the far right and a total row below.
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:28]:
```suggestion
Each card represents an organization and displays runs arranged by workspace and status. 
```

File: docs/dashboard/overview.md [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+
+## Runs per organization
+
+Below the cards displaying total runs by status, run totals are filtered by each organization or your personal workspace. Filtering depends on what you selected in the drop-down options near the top of the page. 
+
+Each card represents an organization. Total runs for the organization are arranged by workspace and status. 
+
+![](_images/dashboard_orgs.png)
+
+There are various navigation options you can go from here if you want to investigate further on your workflows.
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:32]:
```suggestion
Click a run value in the table to navigate to a run list filtered by the status selected. 

Click a workspace name in the table to navigate to a run list filtered by the workspace selected. 
```

File: docs/dashboard/overview.md [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+
+Each card represents an organization. Total runs for the organization are arranged by workspace and status. 
+
+![](_images/dashboard_orgs.png)
+
+Click a run value in the table to navigate to a run list filtered by the status selected. 
+
+Click a workspace name in the table to navigate to a run list filtered by the workspace selected. 
+
+ - On the one hand, you can click on every workspace name to see the workflow runs in the context of that workspace.
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:36]:
```suggestion
```

File: docs/dashboard/overview.md [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+
+Each card represents an organization. Total runs for the organization are arranged by workspace and status. 
+
+![](_images/dashboard_orgs.png)
+
+Click a run value in the table to navigate to a run list filtered by the status selected. 
+
+Click a workspace name in the table to navigate to a run list filtered by the workspace selected. 
+
+ - On the other hand, you can click on any count number in any cell to see the workflow runs in the context of that workspace that are in that specific status (the filters there will be set for you automatically).
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:36]:
```suggestion
```

**Review by llewellyn-sl — APPROVED** [review]
I've committed a number of suggested changes, please confirm the new statements before merging.

**Comment by swampie** [comment]
@mbosio85 to review the numbers

**Review by mbosio85 — CHANGES_REQUESTED** [review]
The 2nd image has the wrong total number becuase it does not coincide with the . In the Figma is now correct (I didn't check all the row and column sums, but this stood out).

These two numbers should be the same 
image

@joaquimgamero can you update the image with the right numbers?

File: docs/dashboard/overview.md [hunk 18]
```diff
@@ -0,0 +1,36 @@
+---
+title: Dashboard Overview
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:2]:
```suggestion
title: Dashboard overview
```

File: docs/dashboard/overview.md [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+headline: 'Dashboard'
+description: 'View run status overview in Tower'
+---
+
+!!! note 
+    This feature is available from Tower v.22.3.
+
+## Overview
+
+From version 22.3, Tower contains a **Dashboard** page that provides an overview of runs in your organizations and personal workspace at a glance. The dashboard is accessed from the user menu in the top right corner. Click your avatar, then select "Dashboard".
```
Comment (llewellyn-sl) [inline docs/dashboard/overview.md:12]:
```suggestion
From version 22.3, Tower contains a **Dashboard** page that provides an overview of runs in your organizations and personal workspace at a glance. The dashboard is accessed from the user avatar menu in the top right corner. Click your avatar, then select "Dashboard".
```

**Review by llewellyn-sl — APPROVED** [review]

**Pull request merged**